src/Components/Artist/ArtistCollectionsRail/ArtistCollectionsRail.tsx

```diff
@@ -55,8 +55,8 @@ export class ArtistCollectionsRail extends React.Component<
           <Carousel
             height={200}
             settings={{
-              slidesToScroll: 1,
-              infinite: true,
+              groupCells: 1,
+              wrapAround: true,
```

Comment:
wondering -- should these two settings be in the defaults?

Reply:
These two settings are unique to this carousel alone.